res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1776,20 +2008,21 @@ PioneerDDJSX.hotCueLeds = function(value, group, control) {
 
 PioneerDDJSX.VuMeterLeds = function(value, group, control) {
     var midiBaseAdress = 0xB0,
-        midiOut = 0;
+        channel = 0x02,
+        midiOut = 0x00;
 
     value = 1 / (1 - PioneerDDJSX.cutVumeter) * (value - PioneerDDJSX.cutVumeter);
```

Comment:
It's a leftover from the DDJ-SB2 mapping and I didn't touch that yet. I thought the same as you. So then I think it's better to delete this part, ok?